projects/spectator/test/with-routing/my-page.component.spec.ts

```diff
@@ -196,4 +196,65 @@ describe('MyPageComponent', () => {
       expect(console.warn).toHaveBeenCalled();
     });
   });
+
+  describe('should support router state mocking', () => {
+    const createComponent = createRoutingFactory({
+      component: MyPageComponent,
+      root: new ActivatedRouteStub({
+        params: { root: '1' }
+      }),
+      parent: new ActivatedRouteStub({
+        params: { parent: '2' }
+      }),
+      children: [
+        new ActivatedRouteStub({
+          params: { child1: '3' }
+        }),
+        new ActivatedRouteStub({
+          params: { child2: '4' }
+        })
+      ],
+      firstChild: new ActivatedRouteStub({
+        params: { firstChild: '5' }
+      })
+    });
+
+    it('should support root mock', () => {
+      const spectator = createComponent();
+
+      let rootParam;
+      spectator.component.root!.paramMap.subscribe(params => {
```

Comment:
Why not writing the expectations inside the `subscribe` block?

Reply:
Good point 👍

Reply:
Fixed